src/stim/cmd/command_help.cc

```diff
@@ -90,6 +91,7 @@ std::vector<SubCommandHelp> make_sub_command_help() {
     help_help.description = "Prints helpful information about using stim.";
     auto result = std::vector<SubCommandHelp>{
         command_analyze_errors_help(),
+        command_convert_help(),
```

Comment:
You need to run the regen docs command in the dev/ directory to include this in the docs in the repo. (It assumes you're in a python virtual env with the dev wheel installed; see the dev docs.)

Reply:
done